src/azure/accountService.ts

```diff
@@ -65,40 +73,17 @@ export class AccountService {
     }
 
     public async createSecurityTokenMapping(): Promise<any> {
-        if (!this._token) {
-            let promise = new Deferred();
-            this._token = this._session.credentials.getToken((error, result ) => {
-                if (result) {
-                    this._isStale = false;
-                    this._token = result;
-                }
-                if (error) {
-                    this._isStale = true;
-                }
-                promise.resolve();
-            });
-            await promise;
-        }
         let mapping = {};
-        mapping[this._session.tenantId] = {
-            expiresOn: this._token.expiresOn.toISOString(),
-            resource: this._token.resource,
-            tokenType: this._token.tokenType,
-            token: this._token.accessToken
+        mapping[this.account.properties.tenants[0].id] = {
```

Comment:
do we always take the first tenant? users can have multiple now right?

Reply:
yeah you're right, let me check what they do in ADS for this.